src/Payments.Mvc/Views/Shared/_LeftNavPartial.cshtml

```diff
@@ -47,40 +48,46 @@
             </div>
             <span class="navitem">Support</span>
         </div>
+    </a>    
+
+    <a class="nav-settings" asp-controller="Teams" asp-action="Details" asp-team="@teamSlug">
```

Comment:
@jpknoll should be asp-route-team= here and in the sub menu choices

But, if they don't select a team, they get errors. As an admin may not be in any teams maybe put this here:
asp-route-team="@(string.IsNullOrWhiteSpace(teamSlug) ? "----" : teamSlug)

Reply:
At some point they'll be required to select a team at login or a default one in their user settings.

Reply:
An admin that sets up teams, may never be in a team...